content/news/events-january-2025.md

```diff
@@ -0,0 +1,82 @@
+
+<br/>
+
+
+### Wednesday Mesh Evening - Wednesday January 8th @ 19:00 - 22:00
+
+Tired of spending time with your family? I know I am. Join us for the first social gathering of the year! Lots of new nodes, the region's Meshtastic coverage is booming - what a good time to mesh around this January is shaping up to be!
+
+Join the network! Explore mesh networking in all its forms. Discuss, research, test the unknown, and deploy peer-to-peer infrastructure!
+
+Everyone is hosting, and **kryma** invites you to host as well. 
+
+The network must grow! The network must grow! The network must grow! The network must grow! The network must grow! The network must grow! The network mu- (the author has been disconnected from the network)
+
+<br/>
+
+
+### Workshop Reorganization Planning! - Thursday January 9th @ 20:00 - 21:30
+
+The Foulab workshop area needs some love! We have a lot of tools, both traditional (hand tools, drill press, bandsaw, air compressor, welder, etc) and modern (3D printer, CNC mills, laser cutter) but not a lot of good places to put them.
+
+We do, however, have an excess of storage racking left over from the museum expansion, so let's solve two problems at once and reorganize the workshop! 
+
+This is the initial planning meeting, to measure out the tools we have, propose racking layouts, and prepare for the work that lies ahead. Anyone interested in helping out is invited to attend! 
+
+<br/>
+
+
+### Open Gaming Night - Tuesday January 21st @ 20:00 - 22:00
+
+Starting in January, we've opened up the monthly Warhammer 40k gaming night to include other games. This month, **der_moderne_man** will bring Steve Jackson's fantasy dueling games Melee & Wizard. Others are encouraged to bring games they want to play. Warhammer? Magic the Gathering? Risk? Mouseguard: Swords & Strongholds? 
+
+The more the merrier! 
+
+<br/>
+
+
+### FOULAB Monthly Meeting - Thursday January 23rd @ 20:00 - 21:30
+
+The first meeting of the year! Make a New Year's resolution to come to this meeting and make your voice heard on things that mattermost. 
+
+Share your ideas, your projects, and your thoughts on the future of Foulab. Your input is important to our future!
+
+<br/>
+
+
+### Project Night - Tuesday January 28th @ 19:00 - 22:00
+
+Project night is back! Did you work on something exciting over the holidays? Bring it to project night and show it off! 
+
+Let your creativity (and ego) shine! Practice your presentation skills, and get feedback from your peers. Talk about the great things you've done, and the challenges you've faced. 
+
+Beginners welcome! If you're new to the space, come by and see what we're all about. It's a great way to meet people and get involved in the community.
+
+<br/>
+
+
+### Killed by Dice Presents: Three Sixteen V - Saturday January 26th @ 16:00 - 21:00
```

Comment:
Saturday is Jan 25 - is this Saturday or Sunday?

Reply:
Good catch. I'll review quickly

Reply:
Issue resolved and changes pushed. No need to update `ics` since that's generated on the date itself. 

ics had only datestamp updated when I checked